app/src/main/java/com/mapbox/maps/testapp/examples/HeatmapLayerActivity.kt

```diff
@@ -29,10 +30,12 @@ class HeatmapLayerActivity : AppCompatActivity() {
     val binding = ActivityHeatmapLayerBinding.inflate(layoutInflater)
     setContentView(binding.root)
 
-    mapboxMap = binding.mapView.getMapboxMap()
-    mapboxMap.loadStyleUri(
-      styleUri = Style.DARK
-    ) { style -> addRuntimeLayers(style) }
+    mapboxMap = binding.mapView.getMapboxMap().apply {
+      loadStyleUri(
+        styleUri = Style.DARK
```

Comment:
Will compiler accept just `::addRuntimeLayers` instead of lambda?

Reply:
it seems that no